README.md

```diff
@@ -195,11 +195,10 @@ An [Effect] are the instruction to the [Executor] to run set of [Task]s concurre
 
 ```ipldsch
 type Task struct {
-  on      URI
-  call    Ability
-  input   {String : Any}
-
-  nnc     string
+  obj    URI
```

Comment:
Yeah, I'd go w/ `rsc` or `using` as `obj` is too overloaded in it's own accord.

Reply:
Can we just call it `uri` or a `url` ? I also think that `obj` is too overloaded